app/views/rubric_criteria/_rubric_criterion_level.html.erb

```diff
@@ -2,26 +2,27 @@
 # Erb template for display rubric criterion levels in the assignment rubric manager
 %>
 <%= criterion_form.fields_for :levels do |level_form| %>
-  <div class="rubric_level_header">
-    <%= I18n.t("rubric_criteria.level.name") + " " + level_form.index.to_s %>
-  </div>
-  <div class="destroy_checkbox">
-    <%= level_form.check_box :_destroy%>
-    <%= level_form.label :_destroy, I18n.t("criteria.delete.checkbox") + " " + level_form.index.to_s %>
-  </div>
-  <div>
-    <%= level_form.label :name, Level.human_attribute_name(:name), class: 'required' %>
-    <%= level_form.text_field :name %>
-  </div>
-  <div>
-    <%= level_form.label :mark, class: 'required' %>
-    <%= level_form.number_field :mark, step: 0.1 %>
-  </div>
-  <div>
-    <%= level_form.label :description, Level.human_attribute_name(:description), class: 'description required' %>
-    <%= level_form.text_area :description %>
+  <div class='level'>
+    <div id = "display_name" class="rubric_level_header">
+      <%= I18n.t("rubric_criteria.level.name") + " " + level_form.index.to_s %>
```

Comment:
You introduced a locale string `rubric_criteria.level.level_index` that you should use here.

Reply:
I wasn't able to find a way to introduce %{level_form.index.to_s} within the string as there is no more index any more so I cannot use level_index. Should I just delete rubric_criteri.level.level_index?

Reply:
It's not clear what you mean by "there is no more index any more". You should be passing an `index` to the `I18n.t` call, e.g. `I18n.t('...', index: ...)`.